### PR TITLE
Add/first views and routes

### PR DESCRIPTION
**Pull request by habundloot**
i left "profile"-view out of store so we dont get an auto generated link in navigation list but a manual btn on top of navigation

### CODE REVIEW

File: src/components/Dashboard/Overview.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
```
Comment (0m3gaC0d3) [inline src/components/Dashboard/Overview.vue:1]: Please only user upercase names for .vue files. "src/components/**D**ashboard/Overview.vue" to "src/components/**d**ashboard/Overview.vue "

File: src/components/Dashboard/Overview.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="row">
+      <div class="col-4">
+        <app-event-overview></app-event-overview>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import AppEventOverview from './../Events/Overview'
```
Comment (0m3gaC0d3) [inline src/components/Dashboard/Overview.vue:12]: Why the App**Event**Overview naming?

File: src/components/Events/Overview.vue [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="events-overview-container">
+    <p>eventsoverview</p>
+  </div>
+</template>
+
+<script>
+
```
Comment (0m3gaC0d3) [inline src/components/Events/Overview.vue:8]: Please remove the empty line.

File: src/components/Events/Overview.vue [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+export default {
+  data () {
+    return {
+      headline: 'Events'
```
Comment (0m3gaC0d3) [inline src/components/Events/Overview.vue:12]: Remove unused data.

File: src/router/index.js [hunk 5]
```diff
@@ -1,16 +1,30 @@
...
     // meta: { requiresAuth: true }
+  },
+  {
+    path: '/events',
+    name: 'Events',
+    component: AppEvents
+    // meta: { requiresAuth: true }
+  },
+  {
+    path: '/my-profile',
```
Comment (0m3gaC0d3) [inline src/router/index.js:24]: Do we really need the "my-" prefix? Think thats redundant.

File: src/components/Dashboard/Overview.vue (hunk 2 above)
Comment (habundloot) [reply src/components/Dashboard/Overview.vue:12]: I had EventOverview and Dashboard Overview first. But maybe Dashboard should be one main component and be fed with minor components. So I changed the name to App**Event**Listing if we get something like "AppPollListing" in the future.

**Pull request merged**